PowerBI.JavaScript.nuspec

```diff
@@ -17,6 +17,7 @@
   <files>
     <file src="dist\powerbi.js" target="content\scripts" />
     <file src="dist\powerbi.min.js" target="content\scripts" />
+    <file src="dist\powerbi.js.map" target="content\scripts" />
     <file src="dist\powerbi.d.ts" target="content\scripts\typings" />
```

Comment:
Is there a way to set this to include the dist directory so we can match how npm works and not have to worry about updating this?

There are more .d.ts files than jsut the powerbi.d.ts now